_events/sr2022/competition.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Competition
+date: 2022-04-23 09:00:00 Europe/London
+layout: event
+type: competition
+location: Southampton University
+---
+
+The Student Robotics Competition marks the ultimate stage of SR2022, showcasing the months of preparation. Our teams will work hard to push their robots to the limit and quickly make repairs between league matches. We will finish the weekend with the finals where the best bots will be named victorious.
+
+## Schedule
+
+The competition will be held from *Saturday 23rd April* until *Sunday 24th April*. Doors open 9am and we aim to finish around 6pm.
+
+## Location
+
+The competition will be held in the Students' Union building at the [University of Southampton's Highfield campus][soton-campus-directions]:
+
+Student's Union
+University of Southampton
+University Road
+Southampton
+SO17 1BJ
+United Kingdom
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2514.4224296392836!2d-1.399433684090786!3d50.934400360256014!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x487473f7089148ab%3A0xaeeea75e9ec72c88!2sUniversity+of+Southampton+Students&#39;+Union+(SUSU)!5e0!3m2!1sen!2suk!4v1551272133022" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+
+## Parking
+
+We have a limited amount of free parking on site for competitors.
+
+![parking information]({{ '/resources/sr2022/parking-map.png' | prepend: site.baseurl }})
+
+## COVID-19
+
+Please see our [covid updates]({{ '/covid-19' | prepend: site.baseurl }}) page.
```

Comment:
Nit: We seem to have always referred to Covid with its full name would be nice to be consistent here.
```suggestion
Please see our [COVID-19 updates]({{ '/covid-19' | prepend: site.baseurl }}) page.
```